src/react-apps/applications/service-development/src/actions/navigationActions/navigationActionTypes.ts

```diff
@@ -0,0 +1,3 @@
+const moduleName: string = 'NAVIGATION';
+
+export const TOGGLE_DRAWER: string = `${moduleName}.TOGGLE_DRAWER`;
```

Comment:
no `TOGGLE_DRAWER_ERROR` action-type?

Reply:
No, modelled this on the `setDesignMode` action in the ux-editor which also just har the one action-type. 
Will remove both rejected and fulfilled actions since no saga is being used.